tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/UserSteps.cs

```diff
@@ -7,29 +18,135 @@ namespace NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Steps
     [Binding]
     internal sealed class UserSteps
     {
+        private const string OrganisationMapDictionary = "OrganisationMapDictionary";
+        private const string AccessTokenKey = "AccessToken";
+        private readonly string _organisationUrl;
+
         private readonly ScenarioContext _context;
+        private Response _response;
+        private Settings _settings;
 
-        public UserSteps(ScenarioContext context)
+        public UserSteps(ScenarioContext context, Response response, Settings settings)
         {
             _context = context;
+            _response = response;
+            _settings = settings;
+            _organisationUrl = settings.OrganisationApiBaseUrl + "/api/v1/Organisations";
         }
 
-        [Given(@"There are Users in the database")]
-        public void GivenThereAreUsersInTheDatabase(Table table)
+        [Given(@"Users exist")]
+        public async Task GivenUsersExist(Table table)
         {
-            var users = table.CreateSet<UserTable>();
+            var users = table.CreateSet<NewUserTable>();
+            foreach (var user in users)
+            {
+                var allOrganisations = _context.Get<IDictionary<string, Guid>>(OrganisationMapDictionary);
+
+                var organisationId = Guid.Empty;
+                if (allOrganisations.ContainsKey(user.OrganisationName))
```

Comment:
I think this could get replaced with this: 
```
organisationId = allOrganisations.TryGetValue(organisationName, out Guid idValue) ? idValue.ToString() : Guid.Empty.ToString();
```